src/main/java/HelpCommand.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Represents a command by the user to display the list of commands available.
+ */
+public class HelpCommand extends Command {
+    /**
+     * Executes the command by displaying the list of commands available.
+     *
+     * @param tasks List of all the tasks of the user.
+     * @param ui Replies the user.
+     * @param storage Stores the new task in the computer.
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        ui.showHelp();
+    }
+
+    /**
+     * Returns false as the command is not an exit command.
+     *
+     * @return False.
+     */
+    @Override
+    public boolean isExit() {
+        return false;
+    }
```

Comment:
A lot of duplicate code because the "isExit" method is overridden for quite a number of that. I suggest to adopt another approach towards handling the exit of the program:)

Reply:
Perhaps can have isExit() to returns false in Command and only override the method in ExitCommand